cmd/erasure-multipart.go

```diff
@@ -122,7 +122,7 @@ func (er erasureObjects) renameAll(ctx context.Context, bucket, prefix string) {
 		wg.Add(1)
 		go func(disk StorageAPI) {
 			defer wg.Done()
-			disk.RenameFile(ctx, bucket, prefix, minioMetaTmpBucket, mustGetUUID())
+			disk.RenameFile(ctx, bucket, prefix, minioMetaTmpDeletedBucket, mustGetUUID())
```

Comment:
Could you change the name of this function to deleteAll() ?

Reply:
> Could you change the name of this function to renameAll() ?

or moveToTrash()

Reply:
Yes will do in subsequent cleanup.